ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/exception/NoContentException.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package eu.elixir.ega.ebi.dataedge.config;
+package eu.elixir.ega.ebi.dataedge.exception;
 
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
```

Comment:
@anandmohan777 I'm using the in the service `ega-data-api-commons` Common > Shared > Config > NotFoundException. Could this work for you? :-) Again, I'm trying to reduce the amount of code and trying to have less duplicates. However, I'm not sure if `NonFound` will work for you so have a look and let me know :)